views/pages/tools/surveys.njk

```diff
@@ -3,29 +3,146 @@
 {% block title %}Survey results{% endblock %}
 
 {% block content %}
+    <h1 class="display-4 mb-4">
+        All survey results
+        <small class="text-muted">{{ surveys.length }} total surveys</small>
+    </h1>
 
-    <h1>All surveys</h1>
     {% for s in surveys %}
-        <h2>{{ s.name }}</h2>
-        <p><strong>Question</strong>: {{ s.question_en }}</p>
+        <nav class="navbar navbar-dark mb-2" style="background-color: #cfddea;">
+            <h2>
+                {{ s.title }}
+                <span class="badge badge-info">{{ s.status }}</span>
+            </h2>
+            <small class="text-muted" title="{{ s.dateCreated }}">Created {{ moment(s.dateCreated.date).fromNow() }}</small>
+        </nav>
+        <p class="lead">
+            {{ s.question }}
+            <span class="badge badge-success">{{ s.winner.percentage }}% said {{ s.winner.title | lower }}</span>
+        </p>
+
+        <div style="height: 450px" class="border mb-4">
+            <canvas id="js-chart-{{ s.id }}" width="400" height="400"></canvas>
+        </div>
+
+        {% set statuses = cycler('success', 'danger') %}
+        <h4 class="mb-2">Response summary</h4>
+        <h6 class="mb-4">
+            {% for choice in s.choices %}
+                {{ choice.title }}
+                <span class="badge badge-{{ statuses.next() }}">{{ choice.responses.length }} votes</span>
+            {% endfor %}
+        </h6>
+
+        <h4>User messages</h4>
         {% for choice in s.choices %}
-            <h3>{{ choice.title_en }}: {{ choice.responses.length }} votes</h3>
-            {% if choice.allow_message %}
-                <ul>
-                    {% for response in choice.responses %}
-                        {% if response.message %}
-                            <li>
-                                "{{ response.message }}" ({{ response.createdAt }})
-                                {% if response.metadata %}
-                                    <pre><code>{{ response.metadata }}</code></pre>
-                                {% endif %}
-                            </li>
-                        {% endif %}
-                    {% endfor %}
-                </ul>
-            {% else %}
-                <p>No additional messages.</p>
+            {% if choice.allowMessage %}
+                <div class="border mb-5" style="max-height: 400px; overflow: auto">
+                    <div class="list-group">
+                        {% for response in choice.responses %}
+                            {% if response.message %}
+                                <div class="list-group-item list-group-item-action flex-column align-items-start">
+                                    <div class="d-flex w-100 justify-content-between">
+                                        <h6 class="mb-3">{{ moment(response.createdAt).format('ddd DD MMM YYYY (hh:mm a)') }}</h6>
```

Comment:
Not a _huge_ fan of moment in the view, rather than creating a filter or something.